docs/api/admin.yaml

```diff
@@ -100,8 +100,8 @@ paths:
     $ref: './paths/admin/{keyspace}~_config~sync.yaml'
   '/{keyspace}/_config/import_filter':
     $ref: './paths/admin/{keyspace}~_config~import_filter.yaml'
-  '/{keyspace}/_resync':
-    $ref: './paths/admin/{keyspace}~_resync.yaml'
+  '/{db}/_resync':
+    $ref: './paths/admin/{db}~_resync.yaml'
```

Comment:
Can you update _resync.yaml to include the POST payload used to specify collections?